content/index.js

```diff
@@ -241,6 +247,27 @@ cli
       options.foldersearch = newFoldersearch;
     }
 
+    // We might need to parse the list of files because it's a string
+    // with a space separator.
+    if (options.files) {
+      // The get-diff-action will make this a massive string that looks like
+      // this: `'content/files/en-us/a/index.html','content/files/en-us/a/index.html'`
+      // so we need to turn that into an array:
+      // ["content/files/en-us/a/index.html", "content/files/en-us/b/index.html"]`
+      // Note, when you use get-diff-action in GitHub Actions, it's a comma
+      // but if you use the manualy `git diff --name-only ...` on your command
+      // line it's a newline.
+      options.files = options.files.split(/[,\n]/).map((item) => {
```

Comment:
Won't this fail on Windows, with its `\r\n` line breaks? It seems safer to do something like:
```suggestion
      options.files = options.files.split(/,|\s+/).map((item) => {
```

Reply:
Would you even get a `\r\n` on Windows?? Also, the only time that would matter is if you wanna debug, on a Windows computer, what's going to happen in Ubuntu. Also, the regexes are not the same:
```
> var options = {files: `'foo','bar buzz','bang'`}
undefined
> options.files.split(/[,\n]/)
[ "'foo'", "'bar buzz'", "'bang'" ]
> options.files.split(/,|\s+/)
[ "'foo'", "'bar", "buzz'", "'bang'" ]
>
```

Reply:
I don't know, just seems like it's possible when debugging on Windows (and of course it'll never happen in CI). Oops, yep it doesn't handle spaces in filenames, but I don't think we should support that anyway. You could do `options.files.split(/,|[\r\n]+/)` instead. It's not a big deal, and we can deal with it if a Windows user complains.